x-pack/test/alerting_api_integration/spaces_only/tests/alerting/find.ts

```diff
@@ -62,6 +63,28 @@ export default function createFindTests({ getService }: FtrProviderContext) {
       expect(Date.parse(match.updatedAt)).to.be.greaterThan(0);
     });
 
+    it('should handle find alert request with get_aggregations appropriately', async () => {
+      const { body: createdAlert } = await supertest
+        .post(`${getUrlPrefix(Spaces.space1.id)}/api/alerts/alert`)
+        .set('kbn-xsrf', 'foo')
+        .send(getTestAlertData())
+        .expect(200);
+      objectRemover.add(Spaces.space1.id, createdAlert.id, 'alert', 'alerts');
+
+      const response = await supertest.get(
+        `${getUrlPrefix(
+          Spaces.space1.id
+        )}/api/alerts/_find?search=test.noop&search_fields=alertTypeId&get_aggregations=true`
+      );
+
+      expect(response.status).to.eql(200);
+      expect(response.body.page).to.equal(1);
+      expect(response.body.perPage).to.be.greaterThan(0);
+      expect(response.body.total).to.be.greaterThan(0);
+      expect(response.body.aggregations).not.to.be(undefined);
+      expect(response.body.aggregations.alertExecutionStatus).not.to.be(undefined);
```

Comment:
I think a comment I made here got dropped (my bad I think), so writing it again ...

I think we should test for non-trivial results here.  Something like, create two alerts, one enabled, one disabled, and wait for the following status result: one "pending" (the disabled one), the other "ok" or "active" (the enabled one).  Make sure there are no alerts running before starting :-).  I think that would be enough to show that the code path is basically working.

Should be easy to find some code to create no-op or always firing alerts; example patterns of doing things like that are here: https://github.com/elastic/kibana/blob/master/x-pack/test/alerting_api_integration/spaces_only/tests/alerting/execution_status.ts

Reply:
I believe I did this here: https://github.com/elastic/kibana/pull/81819/files#diff-3b4b27aa14057f7c0989968902727d1691627dea8461a7871e1cc70769bdb7b8

Let me know if I misunderstood.